docs/cedarling/cedarling-quick-start.md

```diff
@@ -10,14 +10,187 @@ tags:
 
 # Cedarling Quick Start Guide
 
-## This content is in progress
+## Introduction
 
-The Janssen Project documentation is currently in development. Topic pages are being created in order of broadest relevance, and this page is coming in the near future.
+In this guide, we'll demonstrate how to use Cedarling as a sidecar for an API Gateway to authorize requests. 
+This setup involves configuring the Cedarling to protect resources based on enterprise policies.
 
-## Have questions in the meantime?
+### Sequence diagram
```

Comment:
I find the sequence diagram here to be not "quick" -- it's also filled with jargon like "DCR", "bootstrap config", "jwks_endpoint", that a new person might not be familiar with...